CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 ## Master (unreleased)
 
+### Breaking Changes
+
+* Remove `type` param from `status_tag` [#4989][] by [@javierjulio][]
+
+    ```ruby
+    status_tag(status, :ok, class: 'completed', label: 'on')
+    # now changes to:
+    status_tag(status, class: 'completed ok', label: 'on')
```

Comment:
Yes, as someone who has to make this change I now have to go define these styles as part of the upgrade.